static/js/chart/base.ts

```diff
@@ -75,12 +75,17 @@ function getDashes(n: number): string[] {
 }
 
 function getColorFn(labels: string[]): d3.ScaleOrdinal<string, string> {
+  let domain = labels;
   let range;
-  if (labels.length === 1) {
-    range = ["#930000"];
-  } else if (labels.length === 2) {
-    range = ["#930000", "#3288bd"];
+  if (labels.length == 2 && labels[0] == "Female") {
```

Comment:
is this for gender comparison?

Reply:
yes, a hack to get more appropriate colors (otherwise it was an odd shade of purple)